plugins/copr.py

```diff
@@ -141,15 +142,6 @@ def configure(self):
         if os.path.isfile(default_config_file):
             config_files.append(default_config_file)
 
-            copr_plugin_config.read(default_config_file)
-            if copr_plugin_config.has_option('main', 'distribution') and\
-                    copr_plugin_config.has_option('main', 'releasever'):
-                distribution = copr_plugin_config.get('main', 'distribution')
-                releasever = copr_plugin_config.get('main', 'releasever')
-                self.chroot_config = [distribution, releasever]
-            else:
-                self.chroot_config = [False, False]
-
```

Comment:
Hu? So the ` default_config_file` is now not read at all? I think you missed calling `self._get_copr_chroot(default_config_file)`